public/modules/custom/grants_mandate/src/Form/ApplicantMandateForm.php

```diff
@@ -86,10 +86,27 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
     // ],
     // '#required' => TRUE,
     // ];
+    $form['info'] = [
+      '#markup' => '<p>'.$this->t('Choose the applicant role you want to use for the application').'</p>',
+    ];
     $form['actions'] = [
       '#type' => 'actions',
     ];
-    $form['actions']['submit'] = [
+    $form['actions']['registered'] = array(
+      '#type' => 'container',
+      '#attributes' => array('class' => array('hds-card__body')),
+      '#prefix' => '<div class="hds-card hds-card--applicant-role">',
+      '#suffix' => '</div>',
+    );
+    $form['actions']['registered']['info'] = [
```

Comment:
Nää  html stringit ei oo kyllä hyvä pitkän päälle olla tuolla php koodin seassa. PItäs pyrkiä käyttämään templateja ja ylikirjotuksia ja iskee noi htmlkoodit tonne.

Mä oon noita muutellu sitä mukaan kun on tullut eteen, mut ois hyvä tehä suoraan templateitiedostoon tms.

Mä mergeen tän nyt ni päästään eteenpäin ja ku toi kuitenki toimii, mut tähän vois kiinnittää huomiota jatkossa.